test/test_mimic_explainer.py

```diff
@@ -286,33 +286,38 @@ def _timeseries_generated_data(self):
         return X_train, X_test, y_train, y_test, time_column_name
 
     def test_datetime_features(self, mimic_explainer):
-        X_train, _, _, _, _ = self._timeseries_generated_data()
+        X_train, x_test, _, _, _ = self._timeseries_generated_data()
```

Comment:
where does x_test get used?

Reply:
it will be used once this comment is fixed:
# Note: need to fix column names after featurization as more columns are added to surrogate model